geoparquet/environment.yml

```diff
@@ -0,0 +1,18 @@
+name: coguide-geoparquet
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+  - aiohttp
+  # TODO: how to define "fsspec[http]", so we don't have to manually install aiohttp
+  - fsspec
+  - geopandas==0.13.2
+  - ipykernel
+  - jupyterlab
+  - libgdal-arrow-parquet
+  - requests
+  # This is a hack to install pyarrow via pip because libgdal-arrow-parquet constrains libarrow to
+  # v10, which constrains pyarrow to v10, which is too old for the `filesystem` parameter at the end
+  # of the file
+  - pip:
+    - pyarrow
```

Comment:
This hack should be able to be removed after https://github.com/conda-forge/gdal-feedstock/pull/796 is merged